rpcs3/Emu/RSX/RSXFIFO.cpp

```diff
@@ -129,6 +129,10 @@ namespace rsx
 					data = { cmd, 0, m_internal_get };
 					return;
 				}
+
+				// Malformed command, optional recovery
+				data.reg = FIFO_ERROR;
```

Comment:
This caused me pain before and I cannot remember the exact situation. I just remember this used to be here before I moved it down.  Will need testers to ensure I fixed whatever was broken or we end up with tons of "regressions".

Reply:
I can wait a day or two for testers to test the pr if needed.